docs/documentation/cl_options/index.rst

```diff
@@ -0,0 +1,40 @@
+Command Line Arguments
+======================
+
+What starting vimiv form the command line you have the ability to pass a number of
+different argument to vimiv. The general calling structure is depicted below:
```

Comment:
I would leave out the "depicted below", just
"The general calling structure is:"

Reply:
rebased and fixed in 5aa4b56a13f4f7aa76d208b9a10b6e891e525ad6